triton-client/src/store/thunks.ts

```diff
@@ -78,6 +79,29 @@ export const fetchDatasets = (externalProjectId: ExternalProjectID) => async (di
 	}
 }
 
+const updateProjectUsage = (projectId: ExternalProjectID) => async (dispatch: AppDispatch, getState: () => RootState) => {
+	const readsets = Object.values(getState().readsetsState.readsetsById).reduce<TritonReadset[]>((readsets, readset) => {
+		if (readset && readset.dataset && getState().datasetsState.datasetsById[readset.dataset]?.external_project_id === projectId) {
+			readsets.push(readset)
+		}
+		return readsets
+	}, [])
+	const diskUsage: ProjectState['diskUsage'] = {
+		'GLOBUS': 0,
```

Comment:
We should have a constant set in the config.ts SUPPORTED_DOWNLOAD_TYPES so we can stop hardcoding these 2 all over the place ...

Reply:
No need to.
```ts
type DownloadRequestType = "SFTP" | "GLOBUS"
```

Reply:
We just use typescript to limit us ;)